integration-tests/hibernate-orm-mongodb-panache/src/main/java/io/quarkus/it/panache/mongodb/MongoBook.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.it.panache.mongodb;
+
+import org.bson.codecs.pojo.annotations.BsonProperty;
+import org.bson.types.ObjectId;
+
+import io.quarkus.mongodb.panache.MongoEntity;
+
+@MongoEntity(collection = "TheBook", clientName = "cl2")
```

Comment:
Keep it simple, don't use a different clientName

Reply:
Fixed in f470047